tests/cnf/core/network/dpdk/tests/rootless.go

```diff
@@ -261,7 +266,11 @@ var _ = Describe("rootless", Ordered, Label(tsparams.LabelSuite), ContinueOnFail
 					firstInterfaceBasedOnTapOne:  minimumExpectedDPDKRate,
 					secondInterfaceBasedOnTapOne: maxMulticastNoiseRate},
 			)
-			rxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapTwoInterfaceName, "${PCIDEVICE_OPENSHIFT_IO_DPDKPOLICYONE}"))
+
+			err = cmd.RxTrafficOnClientPod(clientPod, defineTestPmdCmd(tapTwoInterfaceName,
+				"${PCIDEVICE_OPENSHIFT_IO_DPDKPOLICYONE}"))
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
```

Comment:
Remove fmt.Sprintf, the error will shown anyway

Reply:
removed